vmaas/load.go

```diff
@@ -410,6 +411,13 @@ func loadRepoDetails(c *Cache) { //nolint: funlen
 			prodID2RepoIDs[det.ProductID] = []RepoID{}
 		}
 		prodID2RepoIDs[det.ProductID] = append(prodID2RepoIDs[det.ProductID], repoID)
+
+		if lastChange != nil {
+			lastChangeTime, err := time.Parse(time.RFC3339, *lastChange)
+			if err == nil {
+				det.LastChange = &lastChangeTime
+			}
+		}
```

Comment:
LastChange is never set in c.RepoDetails
```suggestion
		lastChangeTime, err := time.Parse(time.RFC3339, lastChange)
		if err == nil {
			det.LastChange = &lastChangeTime
		}
		id2repoDetail[repoID] = det
```
and remove `id2repoDetail[repoID] = det` on line 388